### PR TITLE
alpha

### PR DESCRIPTION
**Pull request by tanner0101**

### CODE REVIEW

File: Sources/FluentSQLiteDriver/SQLite+Fluent.swift [hunk 1]
```diff
@@ -142,6 +141,14 @@ extension SQLiteRow: SQLRow {
     }
 }
 
+import Foundation
+
+extension SQLiteError: LocalizedError {
+    public var errorDescription: String? {
```
Comment (broadwaylamb) [inline Sources/FluentSQLiteDriver/SQLite+Fluent.swift]: Why is this added? `self.description` clearly is not localized, and the users of this library (especially those that want to use it on macOS/iOS) might want to add localization themselves.

Comment (tanner0101) [reply Sources/FluentSQLiteDriver/SQLite+Fluent.swift]: @broadwaylamb without this, the errors won't show up correctly if thrown at top-level in an XCTestCase. If there's a better way to achieve this, I'm all for it.

**Comment by penny-coin** [comment]
Hey @tanner0101, you just merged a pull request, have a coin! 

You now have 1247 coins.
